crates/hyperqueue/src/server/rpc.rs

```diff
@@ -56,6 +56,7 @@ impl Backend {
         key: Arc<SecretKey>,
         idle_timeout: Option<Duration>,
         worker_port: Option<u16>,
+        event_store_size: Option<u32>,
```

Comment:
Let's make this `usize` and move the responsibility of choosing the default to the CLI.